src/constants.ts

```diff
@@ -16,6 +16,9 @@ export const ethereum = (window as any).ethereum as MetaMaskInpageProvider;
 
 export const strategyPathKey = 'strategyPath';
 
+export const MAGIC_ADDRESS = '0x539bdE0d7Dbd336b79148AA742883198BBF60342';
```

Comment:
you've extracted the address to `constants` and that's good, but you forgot to clean up after yourself. you know this address is used in more than one place, so you need to use this variable in all of those places. always make sure to do that